src/components/Facets.tsx

```diff
@@ -95,6 +95,14 @@ export function Facets(props: FacetsProps) {
                 facetType = getFacetTypeFromFacet(facet, hierarchicalFieldIds);
               }
 
+              facetProps = {
```

Comment:
I would like to shout out the `useComposedCssClasses` hook in case you haven't seen it yet, which is what we typically use for merging css classes. I don't think it's necessary here because we aren't merging the styling, instead we are using the customCssClasses of a customFacetElement if they are defined and ignoring the classes set at the Facets level. If we wanted to merge the css classes from the Facets and the custom facets, then the `useComposedCssClasses` hook should be used

Reply:
This merges the two objects but I want to have the classes on a singular facet override the classes set on the facets to make sure the styles on the singular facet are used.